From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@Juliako__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (Juliako). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'nitinme-hdi-TOC-REORG' of https://github.com/nitinme/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
fixing validation warnings

nitinme committed:
Fixing some TOC entries

nitinme committed:
Merge pull request #26226 from nitinme/nitinme-hdi-TOC-REORG

Adding R Server TOC

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

Ja-Dunn committed:
Merge branch 'master' of https://github.com/microsoft/azure-docs-pr into service-fabric-get-started-azure-cluster

Ja-Dunn committed:
fixes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

mhko committed:
Add search-api-2016-09-01-preview.md, update the api-version table.

mumian committed:
add general topics

mumian committed:
add domain-joined

mumian committed:
update yml

mumian committed:
add interactive query yml

mumian committed:
merge other cluster types

mumian committed:
fix links

mumian committed:
minor edit

mumian committed:
edit

mumian committed:
incorporate Nitin's feedback

mumian committed:
edit multi-user

nitinme committed:
adding domain-joined cluster type and fixing TOC order

nitinme committed:
Merge pull request #26120 from mumian/newtoc1024

new yml

nitinme committed:
Merge branches 'nitinme-hdi-TOC-REORG' and 'nitinme-hdi-TOC-REORG' of https://github.com/nitinme/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

# Conflicts:
#	articles/hdinsight/TOC.yml

nitinme committed:
Merge pull request #26321 from nitinme/nitinme-hdi-TOC-REORG

Adding domain joined cluster type

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

nitinme committed:
fixing the redirect files

nitinme committed:
Merge pull request #26334 from nitinme/nitinme-hdi-TOC-REORG

Fixing redirect files

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

mhko committed:
Remove search-api-scoring-profiles-2015-02-28-preview.md.
There is a MSDN doc that contains almost identical info.
https://docs.microsoft.com/en-us/rest/api/searchservice/add-scoring-profiles-to-a-search-index

mhko committed:
Remove search-api-indexers-2015-02-28-preview.md
There is an MSDN doc that contains almost identical info.
https://docs.microsoft.com/en-us/rest/api/searchservice/indexer-operations

mhko committed:
Update references to the old preview api version. Update TOC.

mhko committed:
Remove obsolete search-api-2015-02-28-preview.md

mhko committed:
Add an example that illustrates targetting a preview api version.

mhko committed:
Incorporate feedback. Updated the api-version table for the preview SDK.

nitinme committed:
fixing merge conflicts

nitinme committed:
fixing validation error

adegeo committed:
New part in tutorial for scaling

seanli1988 committed:
compatibility article

adegeo committed:
Fixed minor typo and link

adegeo committed:
Minor wording changes

adegeo committed:
Minor wording changes

mhko committed:
Incorporate feedback

nitinme committed:
fixing some more validation errors

nitinme committed:
adding a missing file

nitinme committed:
fixing merge conflict

nitinme committed:
fixing another file path

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into conflicts

Commit:
Updated region availability

seanli1988 committed:
formatting

adegeo committed:
Minor verbage tweaks

seanli1988 committed:
formatting

seanli1988 committed:
formatting

seanli1988 committed:
formatting

tynevi committed:
Merge pull request #26432 from nitinme/conflicts

Fix

mhko committed:
Fix a broken link.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nitinme-hdi-TOC-REORG

v-anpasi committed:
Bringing even with master.

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

tamram committed:
general qs tweaks

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1030

LizCasey committed:
Updated re-direction JSON file for articles being retired.

adegeo committed:
added azure

Commit:
updating TOC to use <clustername> and get started buckets

Commit:
updating toc

nitinme committed:
Making TOCs consistent

nitinme committed:
removing the Apache prefix

nitinme committed:
Merge pull request #26603 from Blackmist/release-hdinsight-toc-reorg

Release hdinsight toc reorg

nitinme committed:
Merge pull request #26604 from nitinme/nitinme-hdi-TOC-REORG

Making TOCs consistent

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

nitinme committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into nitinme-hdi-TOC-REORG

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

dlepow committed:
Batch VNet: UDR updates

tamram committed:
general qs tweaks

LizCasey committed:
Deleted files as part of re-direct effort for task 1103322.

adegeo committed:
minor tweaks

sharonlo101 committed:
fixed bugs in scheduler trigger

v-anpasi committed:
Bringing even with master.

shengcMSFT committed:
Update compute-linked-services.md

fix syntax error on additional linked service

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/release-hdinsight-toc-reorg' into nitinme-hdi-TOC-REORG

mumian committed:
edit yml files

v-viksur committed:
(ForumsTeam) Requesting to edit spelling mistake 

I am requesting to edit spelling from "RESART" to "RESTART"

mumian committed:
remove apache from apache hbase

nitinme committed:
Merge pull request #26654 from mumian/newtoc1024

edit yml files

nitinme committed:
fixing merge conflict

nitinme committed:
fixing paths

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

nitinme committed:
fixing TOC casing

nitinme committed:
Fixing TOC

dlepow committed:
Screenshot border

Ja-Dunn committed:
Update service-fabric-get-started-azure-cluster.md

deleted sentence per Ryan

iainfoulds committed:
New auto OS upgrade doc

iainfoulds committed:
Adding new doc to TOC

iainfoulds committed:
Fixing link validation error

seanli1988 committed:
formatting

ktoliver committed:
Update app-insights-profiler.md

more edits

seanli1988 committed:
toc

tynevi committed:
Merge pull request #26667 from nitinme/nitinme-hdi-TOC-REORG

Fixing some conflicts and other updates

seanli1988 committed:
formatting

tamram committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tamram-1030

ktoliver committed:
Update app-insights-profiler.md

tamram committed:
quickstart include for account creation

mmacy committed:
[acr] geo-rep tutorial portal png update

Commit:
quickfix02

mhko committed:
Incorporate suggetsions.

HeidiSteen committed:
changed search analytics to lexical analysis

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

mmacy committed:
[acr] app services > web apps for containers

spelluru committed:
freshness review

spelluru committed:
merge conflict

dlepow committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into fix

tamram committed:
quickstart include for account creation

spelluru committed:
freshness review

spelluru committed:
freshness review

spelluru committed:
freshness review

dlepow committed:
Pradeep review comments

dlepow committed:
Pradeep review comments

dlepow committed:
fixed link

nitinme committed:
Merge conflict fixes and other updates

# Conflicts:
#	articles/data-lake-store/data-lake-store-data-operations-net-sdk.md
#	articles/data-lake-store/data-lake-store-data-operations-python.md

#	articles/data-lake-store/data-lake-store-end-user-authenticate-net-sdk.md

#	articles/data-lake-store/data-lake-store-end-user-authenticate-python.md
#	articles/data-lake-store/data-lake-store-get-started-net-sdk.md
#	articles/data-lake-store/data-lake-store-get-started-python.md

#	articles/data-lake-store/data-lake-store-service-to-service-authenticate-net-sdk.md

#	articles/data-lake-store/data-lake-store-service-to-service-authenticate-python.md

#	articles/data-lake-store/data-lakes-store-authentication-using-azure-active-directory.md

neilpeterson committed:
updated video

seanli1988 committed:
formatting

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-freshness-updates

Commit:
Changed routing heading.

seanli1988 committed:
formatting

MicrosoftGuyJFlo committed:
Update for freshness Admin password reset doc

Commit:
Merge pull request #26690 from MarkusVi/quickfix02

quickfix02

ShawnJackson committed:
Update app-insights-profiler.md

ktoliver committed:
Merge pull request #26712 from nitinme/nitinme-freshness-updates

Data Lake Store updates

iainfoulds committed:
Update TOC.md

Commit:
task 394: add instructions for appending encrypted token

ktoliver committed:
Merge pull request #26446 from seanli1988/serverparam

compatibility article

ShawnJackson committed:
Merge pull request #26095 from ktoliver/app-insights-profiler

edit pass: app-insights-profiler.md

adegeo committed:
minor tweaks

ktoliver committed:
Merge pull request #26689 from mmacy/acr-geo-rep-tutorial-update-01

[acr] geo-rep tutorial portal png update

ktoliver committed:
Merge pull request #26713 from MicrosoftGuyJFlo/FreshnessAdminPasswordReset

Update for freshness Admin password reset doc

ecfan committed:
Remove references to classic portal

rwike77 committed:
cert info

Commit:
Merge pull request #26716 from dksimpson/dean-t394

task 394: add instructions for appending encrypted token

rwike77 committed:
updated formatting

tynevi committed:
Merge pull request #26428 from MicrosoftDocs/release-hdinsight-toc-reorg

Release hdinsight toc reorg

rwike77 committed:
updated formatting

Commit:
redirects for 3 obsolete VM topics to be retired

ShawnJackson committed:
Update service-fabric-get-started-azure-cluster.md

rmca14 committed:
Merge pull request #26263 from mhko/master

Add search-api-2016-09-01-preview.md, update the api-version table.

rwike77 committed:
more cert info

ShawnJackson committed:
Merge pull request #26249 from Ja-Dunn/service-fabric-get-started-azure-cluster

edit pass: service-fabric-get-started-azure-cluster

rwike77 committed:
formatting

christiankuhtz committed:
tweak

cychua committed:
Add error case for missing policy

ramankumarlive committed:
added a section for Managed Disks migration

added a section for Managed Disks migration

Commit:
retiring 3 obsolete vm provisioning topics

ramankumarlive committed:
updated text

Commit:
Merge pull request #26722 from cychua/master

Add error case for missing policy

rwike77 committed:
added cert info

Commit:
task 394: edits for ec_ref_allow

Commit:
Merge pull request #26724 from ramankumarlive/patch-9

added a section for Managed Disks migration

PRMerger17 committed:
Merge pull request #26727 from dksimpson/dean-t394

task 394: edits for ec_ref_allow

LizCasey committed:
Merge branch 'master' into v-elicas1031

Commit:
Merge pull request #26705 from spelluru/adffreshness1101-9

freshness review

Commit:
Merge pull request #26703 from spelluru/adffreshness1101-8

freshness review

PRMerger10 committed:
Merge pull request #26706 from spelluru/adffreshness1101-10

freshness review

spelluru committed:
freshness review

JasonWHowell committed:
Adding firewall screenshot

PRMerger14 committed:
Merge pull request #26721 from christiankuhtz/patch-31

tweak

JasonWHowell committed:
Adding firewall screenshot

Commit:
task 394: edits for image hover text

Lisaco88 committed:
Merge pull request #26718 from ecfan/remove-classic-portal

Remove references to classic portal

Commit:
Fixing pre-reqs

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into javaStorageQS

Lisaco88 committed:
Merge pull request #26719 from rwike77/sfctl

Sfctl formatting updates

PRMerger14 committed:
Merge pull request #26732 from dksimpson/dean-t394

task 394: edits for image hover text

PRMerger15 committed:
Merge pull request #26733 from roygara/javaStorageQS

Updating pre-reqs intro to have better formatting

Lisaco88 committed:
Merge pull request #26725 from bradsev/retiring-obsolete-vm-topics

retiring 3 obsolete vm provisioning topics

Lisaco88 committed:
Merge pull request #26720 from bradsev/redirect-for-obsolete-vm-topics

redirects for 3 obsolete VM topics to be retired

Lisaco88 committed:
Merge pull request #26676 from iainfoulds/patch-4

VMSS auto OS upgrade

Lisaco88 committed:
Merge pull request #26560 from tamram/tamram-1030

general qs tweaks

Lisaco88 committed:
Merge pull request #26710 from jimdial/patch-19

Changed routing heading.

Lisaco88 committed:
Merge pull request #26729 from JasonWHowell/UsabilityStudy

MySQL: Adding firewall screenshot

linda33wj committed:
Fix a typo

PRMerger20 committed:
Merge pull request #26734 from linda33wj/master

Fix a typo

neilpeterson committed:
updated video

PRMerger18 committed:
Merge pull request #26711 from neilpeterson/aks-video

updated video

PRMerger20 committed:
Merge pull request #26726 from rwike77/cluster-tutorial

Cluster tutorial updates

tamram committed:
Update storage-quickstart-blobs-python.md

Fix heading casing, per PR reviewer feedback on #26560.

rwike77 committed:
Moving graphic

PRMerger17 committed:
Merge pull request #26736 from rwike77/patch-15

Moving graphic

PRMerger20 committed:
Merge pull request #26735 from MicrosoftDocs/tamram-patch-4

Update storage-quickstart-blobs-python.md

shengcMSFT committed:
Update data-factory-compute-linked-services.md

Update the latest HDI version support

cephalin committed:
livefyre

dominicbetts committed:
Fix repo name

PRMerger15 committed:
Merge pull request #26742 from dominicbetts/fix-faq

Fix repo name

PRMerger14 committed:
Merge pull request #26468 from jimdial/patch-16

Updated region availability

PRMerger12 committed:
Merge pull request #26741 from cephalin/feedback

livefyre

dominicbetts committed:
Change Enterprise to Standard

Commit:
freshness

Commit:
replaced images

PRMerger14 committed:
Merge pull request #26743 from dominicbetts/enterprise-standard

Change Enterprise to Standard

cychua committed:
REf customers to EXO doc for supported topologies

Commit:
removing link to deprecated document

shengcMSFT committed:
Update data-factory-compute-linked-services.md

tfitzmac committed:
revised images

Commit:
fix region of interest image

tfitzmac committed:
updated date

mimig1 committed:
Cosmos DB: Corrected milliseconds->count

cynthn committed:
Update scheduled-events.md

spelluru committed:
freshness review

spelluru committed:
freshness review

spelluru committed:
freshness review

mimig1 committed:
Cosmos DB: Updated Table API question

PRMerger15 committed:
Merge pull request #26697 from spelluru/adffreshness1101-5

freshness review

PRMerger18 committed:
Merge pull request #26748 from mimig1/patch-154

Cosmos DB: Corrected milliseconds->count

Commit:
Merge pull request #26749 from cynthn/master

Update scheduled-events.md

Commit:
Merge pull request #26746 from cychua/master

REf customers to EXO doc for supported topologies

rjagiewich committed:
Merge pull request #26643 from shengcMSFT/patch-8

Update compute-linked-services.md

Lisaco88 committed:
Merge pull request #26738 from shengcMSFT/patch-10

Update data-factory-compute-linked-services.md

tdykstra committed:
fix cosmos db example

rjagiewich committed:
Merge pull request #26745 from swhite-msft/scottwhi-bing-update-pics

replaced images

PRMerger18 committed:
Merge pull request #26636 from sharonlo101/shlo10-31

fixed bugs in scheduler trigger

Commit:
Merge pull request #26750 from spelluru/adcfreshness1102

freshness review

Commit:
Merge pull request #26751 from spelluru/adcfreshness1102-2

freshness review

Commit:
Merge pull request #26753 from spelluru/adcfreshness1102-3

freshness review

PRMerger16 committed:
Merge pull request #26752 from mimig1/patch-155

Cosmos DB: Updated Table API question

Lisaco88 committed:
Merge pull request #26744 from Blackmist/freshness

freshness

Lisaco88 committed:
Merge pull request #26747 from tfitzmac/1102managed

revised images

PRMerger15 committed:
Merge pull request #26658 from v-viksur/patch-9

(ForumsTeam) Requesting to edit spelling mistake

Lisaco88 committed:
Merge pull request #26594 from LizCasey/v-elicas1031

Updated re-direction JSON file for articles being retired.

tdykstra committed:
fix typo

Lisaco88 committed:
Merge pull request #26444 from Thraka/sftutorial

New part in tutorial for scaling

alkohli committed:
fixed verbiage

Lisaco88 committed:
Merge pull request #26610 from dlepow/fix

Batch VNet: UDR updates

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger20 committed:
Merge pull request #26755 from tdykstra/patch-6

fix cosmos db example

Commit:
task 394: Acrolinx edits

HeidiSteen committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into heidist-master

HeidiSteen committed:
art files need prepublish prior to index.yml update

PRMerger13 committed:
Merge pull request #26760 from alkohli/patch-149

fixed verbiage

sethmanheim committed:
Freshness update and test

PRMerger12 committed:
Merge pull request #26692 from HeidiSteen/heidist-master

Change "search analytics" to "Lexical analysis" in Overview doc

v-alje committed:
Merge pull request #26762 from MicrosoftDocs/FromPublicRepo

From public repo

Commit:
Update cdn-token-auth.md

twooley committed:
new release notes

PRMerger18 committed:
Merge pull request #26764 from dksimpson/dean-t394

task 394: Acrolinx edits

PRMerger16 committed:
Merge pull request #26766 from sethmanheim/relay

Freshness update and test

PRMerger13 committed:
Merge pull request #26767 from twooley/twrelupdate

new release notes

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into FromPrivateRepo
